libadikted/Makefile

```diff
@@ -94,13 +96,13 @@ make-directories:
 	$(MKDIR) bin obj
 
 clean:
-	$(RM) $(OBJ) $(BIN) $(BIN:$(LIBEXT)=$(OTHER_LIBEXT)) $(RES)
+	$(RM) $(OBJ) $(BIN) $(BIN:$(LIBEXT)=$(OTHER_LIBEXT)) #$(RES)
 
-$(BIN): $(OBJ) $(RES)
-	$(LINK) $(if $(filter yes, $(DEBUGME)),$(LFLAGS_DEBUG),$(LFLAGS)) $(LIBS) -o $@ $(OBJ) $(RES)
+$(BIN): $(OBJ) #$(RES)
+	$(LINK) $(if $(filter yes, $(DEBUGME)),$(LFLAGS_DEBUG),$(LFLAGS)) $(LIBS) -o $@ $(OBJ) #$(RES)
 
 obj/%.o: %.c
 	$(CC) $(if $(filter yes, $(DEBUGME)),$(CFLAGS_DEBUG),$(CFLAGS)) $< -o $@
 
 $(RES): $(RC)
-	$(WINDRES) -i $< --input-format=rc -o $@ -O coff
+	#$(WINDRES) -i $< --input-format=rc -o $@ -O coff
```

Comment:
Intentional?

Reply:
Intentional, "windres" is not supported for ELF files (Linux target). Updated code to build $(RES) conditionally under Windows.